### PR TITLE
chore: Bump dependencies

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: requirements/base.in
Comment (JohanLorenzo) [inline requirements/base.in:23]: This is the only dependency I had to pin. It cause this issue[1]:


```
Installed /usr/local/lib/python3.11/site-packages/balrog-3.53-py3.11.egg
Processing dependencies for balrog==3.53
error: [Errno 2] No such file or directory: '/usr/local/lib/python3.11/site-packages/setuptools/_vendor/importlib_resources-6.4.0.dist-info/METADATA'
error building image: error building stage: failed to execute command: waiting for process to exit: exit status 1
```

The rest went smoothly.

[1] https://firefox-ci-tc.services.mozilla.com/tasks/AlIPp-Y4RpqsNxJ-tK1GGQ/runs/1/logs/public/logs/live.log#L2243

**Review by bhearsum — CHANGES_REQUESTED** [review]
sigh, it's unfortunate to see more python dependency tooling bustage.

Also, we don't make regularly UI upgrades because we have virtually zero tests for them. Please remove that :)

**Review by bhearsum — APPROVED** [review]

**Comment by JohanLorenzo** [comment]
I'm removing my review request to @gabrielBusta because I already got a review from @bhearsum and there's no added value to having another pair of eyes.

**Pull request merged**
